### ISSUE TITLE
Error message says 'omfile: either the "file" or "dynfile" parameter must be given'

### ISSUE THREAD

**Issue by fbicknel** [rsyslog/rsyslog#2975]
### Expected behavior
Error message says 'omfile: either the "file" or "dyn**a**file" parameter must be given'
### Actual behavior
Error message says 'omfile: either the "file" or "dynfile" parameter must be given'
#### Setup
Try:
```
template(name="DynoFile" type="string" string="/var/log/remote/%HOSTNAME%/syslog")
ruleset(name="remote") {
    action(type="omfile" dynFile="DynoFile")
}
```
(Label of template irrelevant: hence purposefully and flippantly misspelled.)

Note the spelling of **dynFile**
#### Unexpected Results
`Aug 31 12:01:50 bick-ubtu3 rsyslogd-2211: omfile: either the "file" or "dynfile" parameter must be given [v8.16.0 try http://www.rsyslog.com/e/2211 ]`
#### Change to Fix
```
template(name="DynoFile" type="string" string="/var/log/remote/%HOSTNAME%/syslog")
ruleset(name="remote") {
    action(type="omfile" dynaFile="DynoFile")
}
```
Note the spelling of **dynaFile**
### Environment
- rsyslog version: 8.16.0
- platform: Ubuntu 16.04
- for configuration questions/issues, include rsyslog.conf and included config files

### Why it's important
Because noobs like me spend an hour looking for documentation to explain why this thing doesn't work as expected before finally finding this silly error message led me down the primrose path.

**Comment by fbicknel** [comment]
Pull request [2977](https://github.com/rsyslog/rsyslog/pull/2977) created that fixes this.

**Comment by rgerhards** [comment]
closed via #2977
